### ISSUE TITLE
sh: 1: [[: not found

### ISSUE THREAD

**Issue by axzx** [shopware/shopware#443]
**Description**  

run
` ./psh.phar docker:start`
then
```
(2/3) Starting
> if [[ -n "" ]]; then docker-sync start && echo "\n docker-sync is initially indexing files. It may take some minutes, until code changes take effect"; fi
        sh: 1: [[: not found

(3/3) Starting
```

**Comment by keulinho** [comment]
looks like the script get's executed with shell, but the shebang at the start of the file tells that it should be executed with bash.
Please make sure that you have bash installed and it's up to date.

Try to run following command: `/usr/bin/env bash -c "type [["`, it should print that `[[ is a reserved keyword`.

**Comment by axzx** [comment]
@keulinho 
I installed according to the instructions https://github.com/shopware/platform#quickstart--installation
There's a docker used, so I expect it all to work right away.

**Comment by keulinho** [comment]
@axzx,
this script runs on your host machine to start the docker container, so you need bash installed on your local machine.

It is stated in the [installation requirements](https://docs.shopware.com/en/shopware-platform-dev-en/getting-started/requirements#docker) for the docker set up.

Kind regards
Jonas

**Comment by axzx** [comment]
I work on linux, bash is installed

**Comment by keulinho** [comment]
What's the result of running `/usr/bin/env bash -c "type [["` on your local machine?

**Comment by axzx** [comment]
`[[ jest słowem kluczowym powłoki`
in en
`[[ is the keyword of the shell`

**Comment by keulinho** [comment]
That's really weird as the docker start script first runs the check permission script that also uses the `[[` syntax and there it seems to work.
And both scripts have the same shebang.

**Comment by taltholtmann** [comment]
We fixed this behaviour for the 6.1 Branch in the development template. Psh seems to ignore the shebang. It will soon be merged to master branch.

**Comment by pweyck** [comment]
It seems link psh does not use the https://en.wikipedia.org/wiki/Shebang_(Unix) . So it's likely executing with the default `/bin/sh` which could be a posix shell, that does not understand the bash syntax. To test it you could run: 
```sh
/bin/sh -c '[[ -n "" ]]'
```
